src/components/NavBar/navBarStyle.css

```diff
@@ -0,0 +1,15 @@
+nav {
```

Comment:
Another super nit-picky comment:

It may be a good idea to add unique classNames to the places that we are styling. I am always really cautious of any pollution of the "global namespace" because it can lead to tricky CSS bugs down the road. 

For the `nav` and `a`, we could give each classNames just refer to them by those or by `tag.some-class-name`. For the `.active` className that's part of react-router-dom, we could add on `.active` to make it `tag.some-class-name.active`

examples:
```javascript
// in React component
<NavLink className="nav-link" />
```
```css
// in CSS stylesheet
a.nav-link {
...
}

a.nav-link.active {
...
}
```

Reply:
Thanks @MikeBLambert, we welcome all suggestions to do a better job.  We are going to work on them.